File: src/main/java/com/trip_service/trip_service/route/repositories/RouteRepository.java
package com.trip_service.trip_service.route.repositories;

import com.trip_service.trip_service.route.models.Route;
import org.springframework.data.jpa.repository.JpaRepository;

import java.util.Optional;
import java.util.UUID;

public interface RouteRepository extends JpaRepository<Route, UUID> {

    Optional<Route> findByRouteName(String routeName);

}

File: src/main/kotlin/com/trip_service/trip_service/core/routes/repositories/RouteRepository.kt
package com.trip_service.trip_service.core.routes.repositories

import com.trip_service.trip_service.core.routes.models.Routes
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository
import java.util.UUID

@Repository
interface RouteRepository : JpaRepository<Routes, UUID> {
}